Convex_hull_3/doc/Convex_hull_3/CGAL/convex_hull_3.h

```diff
@@ -65,4 +65,31 @@ void convex_hull_3(InputIterator first, InputIterator last,
 Object& ch_object,
 const Traits& ch_traits = Default_traits);
 
+
+/*!
+\ingroup PkgConvexHull3Functions
+
+\brief computes the convex hull of the set of points in the range.
+It puts in out the points from `range` that are on its convex hull.
+
+\tparam InputRange a range of `Traits::Point_3`, model of `Range`.
```

Comment:
Range::const_iterator is model of what? does the range need to be mutable or ConstRange is fine?